.circleci/config.yml

```diff
@@ -36,7 +36,7 @@ check_dirty_step: &check_dirty_step
 start_android_emulator_step: &start_android_emulator_step
   run:
     name: Start Android emulator
-    command: ./Support/Scripts/start-android-emulator.sh
+    command: ./Support/Scripts/start-android-emulator
```

Comment:
Just to be consistent with the rest, you should name it `start-android-emulator.py`.